kie-spring/src/test/java/org/kie/spring/CodeVersion.java

```diff
@@ -18,6 +18,6 @@
 public class CodeVersion {
 
     // Do not modify this: this is automatically incremented by the maven replacer plugin
-    public final static String VERSION = "7.0.0-SNAPSHOT";
+    public final static String VERSION = "7.1.0-SNAPSHOT";
```

Comment:
according to comment above not sure if we should commit this file ... @psiroky any thoughts?